core/controlplane/config/templates/cluster.yaml

```diff
@@ -775,6 +775,9 @@ worker:
 # See https://github.com/kubernetes-incubator/kube-aws/issues/208 for more information
 #elasticFileSystemId: fs-47a2c22e
 
+# Create shared persistent volume
+#sharedPersistentVolume: false
```

Comment:
Just wondering/Not intended to scope-creep this PR; Are the any use-cases that lead us want to create multiple of shared persistent volumes, possibly varying volume names and storage sizes?

```yaml
sharedPersistentVolumes:
- name: shared-efs
  provider: efs
  size: 500Gi
- name: shared-ebs
  provider: ebs
  size: 100Gi
```